lib/livebook/hubs/dockerfile.ex

```diff
@@ -9,14 +9,16 @@ defmodule Livebook.Hubs.Dockerfile do
           deploy_all: boolean(),
           docker_tag: String.t(),
           clustering: nil | :auto | :dns,
-          zta_provider: atom() | nil
+          zta_provider: atom(),
+          environment_variables: nil | %{String.t() => String.t()}
```

Comment:
Can it default to an empty map instead, so we don't need to deal with `nil`s?

Reply:
Or maybe we always pass it as a list, since we are converting them to lists anyway?